src/explore-education-statistics-admin/src/modules/find-statistics/components/ReleaseHeadlines.tsx

```diff
@@ -35,31 +32,43 @@ const ReleaseHeadlines = ({ release }: Props) => {
         body: '',
       },
     });
-  }, [release.id, release.headlinesSection.id, addContentSectionBlock]);
+  }, [actions, release.id, release.headlinesSection.id]);
```

Comment:
As `actions` is just an object (and isn't memoized), this will cause this callback to inefficiently change on every render. We need to either:

- Memoize `actions` using `useMemo`
- Define the `useCallback` deps as `actions.addContentBlock` rather than `actions`.

Same with all the other `useCallback` deps in this component (and their relevant action).